src/microbe_stage/Spawners.cs

```diff
@@ -102,109 +102,21 @@ public static void GiveFullyGrownChanceForMulticellular(Microbe microbe, Random
     // TODO: this is likely a huge cause of lag. Would be nice to be able
     // to spawn these so that only one per tick is spawned.
     public static IEnumerable<Microbe> SpawnBacteriaColony(Species species, Vector3 location,
-        Vector3 playerPosition, Node worldRoot, PackedScene microbeScene,
```

Comment:
With the playerPosition no longer being in here, does this method still make sure it doesn't spawn the colony members so far away that they can despawn instantly?

Reply:
No, that PR was a victim of bad timing. Colonies now have a max size of 2, and spawn well within the despawn radius.